classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
```

Comment:
- [ ] Please refactor this property to pluralized form. That should be `students` instead of `student`.
```suggestion
  attr_reader :students
```